### PR TITLE
Corrected spelling and Grammar

### PR DESCRIPTION
**Pull request by JJ173-zz**
Replaced `speed2` with `Training Officer` and corrected grammar and spelling mistakes. Please review these changes before I make the voices.

### CODE REVIEW

File: maps/FAF_TUT_Theta_BO/FAF_TUT_Theta_BO_strings.lua [hunk 1]
```diff
@@ -65,52 +61,52 @@ StartBuildOrder = {
...
-    {text = '[speed2]: Be sure to place move order closer to the Mass Extractors to utilize full build range of your ACU. Build range is the small yellow circle.', vid = '', bank = '', cue = '', faction = 'UEF', delay = 5},
-    {text = '[speed2]: Factory will build 3 engineers. One for building your base, 2 for expanding. Then 2 tanks and 2 scouts to protect your expanding enginners and/or for raiding enemy engineers.', vid = '', bank = '', cue = '', faction = 'UEF'},
-    {text = '[speed2]: After that you can build additional engineers and tanks. On a small map like Theta you should not build only engineers from your fist factory since you will never need so many of them.', vid = '', bank = '', cue = '', faction = 'UEF'},
-    {text = '[speed2]: Changing factory rally point will also affect which way the units will leave the factory, you can save some time by that.', vid = '', bank = '', cue = '', faction = 'UEF'},
+    {text = '[Training Officer]: Land Factories are built first, followed by a single Power Generator. Then 4 Mass Extractors along the way to the Hydrocarbon.', vid = '', bank = '', cue = '', faction = 'UEF'},
```
Comment (speed2CZ) [inline maps/FAF_TUT_Theta_BO/FAF_TUT_Theta_BO_strings.lua]: This is the beginning of the build order so you do place just a single Land Factory. So previous was correct

File: maps/FAF_TUT_Theta_BO/FAF_TUT_Theta_BO_strings.lua [hunk 2]
```diff
@@ -65,52 +61,52 @@ StartBuildOrder = {
...
+    {text = '[Training Officer]: Be sure to place move orders closer to the Mass Extractors to utilize the full build range of your ACU. Build range is the small yellow circle on your HUD surrounding the ACU.', vid = '', bank = '', cue = '', faction = 'UEF', delay = 5},
+    {text = '[Training Officer]: The first Land Factory will build 3 engineers. One for building your base and 2 for expanding. Then 2 tanks and 2 scouts to protect your expanding enginners and/or for raiding enemy engineers.', vid = '', bank = '', cue = '', faction = 'UEF'},
+    {text = '[Training Officer]: After that you can build additional engineers and tanks. On a small map like Theta you should not build only engineers from your first factory since you will never need so many of them.', vid = '', bank = '', cue = '', faction = 'UEF'},
+    {text = '[Training Officer]: Changing factory rally points will also affect which way the units will leave the factory, you can save some time by doing that.', vid = '', bank = '', cue = '', faction = 'UEF'},
```
Comment (speed2CZ) [inline maps/FAF_TUT_Theta_BO/FAF_TUT_Theta_BO_strings.lua]: Each factory has only one rally point right? so "Changing factory rally point" is correct

File: maps/FAF_TUT_Theta_BO/FAF_TUT_Theta_BO_strings.lua [hunk 3]
```diff
@@ -65,52 +61,52 @@ StartBuildOrder = {
...
 }
 
 FirstEngineer = {
-    {text = '[speed2]: First engineer will go to build a Hydrocarbon and then several Land Factories around the Mass Extractor. The ACU will assist it until the Hydrocarbon and at least one more factory are done.', vid = '', bank = '', cue = '', faction = 'UEF'},
-    {text = '[speed2]: Always try to place Land Factories around Mass Extractors and Air Factories around Power Generators to fully use the adjacency bonus (cost discount).', vid = '', bank = '', cue = '', faction = 'UEF'},
+    {text = '[Training Officer]: You first engineer will go to build a Hydrocarbon and then several Land Factories around the Mass Extractors. The ACU will assist it until the Hydrocarbon and at least one more factory are done.', vid = '', bank = '', cue = '', faction = 'UEF'},
```
Comment (speed2CZ) [inline maps/FAF_TUT_Theta_BO/FAF_TUT_Theta_BO_strings.lua]: In this case it's building 4 land factories around a single mex so just one "Mass Extractor"

**Review by speed2CZ — CHANGES_REQUESTED** [review]
Fix this few things and I think we're good to go

File: maps/FAF_TUT_Theta_BO/FAF_TUT_Theta_BO_strings.lua (hunk 3 above)
Comment (TheKeyBlue) [reply maps/FAF_TUT_Theta_BO/FAF_TUT_Theta_BO_strings.lua]: `Your first engineer` ?

**Review by speed2CZ — APPROVED** [review]

**Pull request merged**